examples/all_features/assertion_macros.cpp

```diff
@@ -191,3 +191,16 @@ static void someAssertsInFunction() {
 TEST_CASE("some asserts used in a function called by a test case") {
     someAssertsInFunction();
 }
+
+TEST_CASE("macro return values") {
+    int a = 4;
+    int b = 2;
+    DOCTEST_MSVC_SUPPRESS_WARNING_WITH_PUSH(4702)
+    if (CHECK(a == b)) { MESSAGE("should not be reached!"); }
```

Comment:
All the checks return false. Need to duplicate them and cover the case when the return value of CHECK is true. Otherwise possible error when check always returns false is not covered.

Reply:
Already covered in:
- https://github.com/doctest/doctest/blob/dev/examples/all_features/assert_returns_disabled.cpp
and
- https://github.com/doctest/doctest/blob/dev/examples/all_features/assert_returns_disabled_evaluate.cpp